applications/irda/irda-app-file-parser.cpp

```diff
@@ -195,34 +222,81 @@ const char* find_first_not_of(const char* str, char symbol) {
     return str;
 }
 
+static int remove_args32(std::string_view& str, size_t num) {
```

Comment:
There is no reason for passing sting_view by reference. See https://stackoverflow.com/questions/58410793/passing-stdstring-view-by-reference discussion

Reply:
I thought about it for a while and came to this:
I understand that passing by ref has no profit in speed/size as far as sv is a pointer+size.
But I want to return length of removed symbols, so passing by ref is valid for this case.

There can be another approach - pass by value, and then move sv data ptr (remove_prefix) by value of returned value.
But this seems little bit odd as we move this ptr inside function, and then move it again by remove_prefix outside.

So for this case I think this is a valid function signature.